language/stdlib/modules/LibraVMConfig.move

```diff
@@ -40,19 +40,23 @@ module LibraVMConfig {
         /// The cost per-byte written to storage.
         global_memory_per_byte_write_cost: u64,
 
-        /// We charge one unit of gas per-byte for the first 600 bytes
+        /// The flat minimum amount of gas required for any transaction.
+        /// Charged at the start of execution.
         min_transaction_gas_units: u64,
 
-        /// Any transaction over this size will be charged `INTRINSIC_GAS_PER_BYTE` per byte
+        /// Any transaction over this size will be charged `intrinsic_gas_per_byte` units of gas per byte.
         large_transaction_cutoff: u64,
 
-        /// The units of gas that should be charged per byte for every transaction.
+        /// The units of gas that to be charged per byte over the `large_transaction_cutoff` in addition to
+        /// `min_transaction_gas_units` for transactions whose size exceeds `large_transaction_cutoff`.
         instrinsic_gas_per_byte: u64,
```

Comment:
I just noticed that this field is misspelled. It should be "intrinsic" not "instrinsic".

Reply:
good spot!